src/common/json_output.rs

```diff
@@ -35,20 +39,27 @@ pub struct EnclaveDescribeInfo {
     #[serde(rename = "Flags")]
     /// The bit-mask which provides the enclave's launch flags.
     pub flags: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(flatten)]
+    /// Build measurements containing PCRs
+    pub build_info: Option<EnclaveBuildInfo>,
 }
 
 impl EnclaveDescribeInfo {
     /// Create a new `EnclaveDescribeInfo` instance from the given enclave information.
     pub fn new(
+        enclave_name: Option<String>,
         enclave_id: String,
         enclave_cid: u64,
         cpu_count: u64,
         cpu_ids: Vec<u32>,
         memory_mib: u64,
         state: String,
         flags: String,
+        build_info: Option<EnclaveBuildInfo>,
```

Comment:
This seems to be part of another patch.

Reply:
This is added so the output of the `describe-enclaves` includes the new information. Should this be part of another commit or another PR?

Reply:
Whichever you think makes more sense. I prefer smaller/more PRs.

Reply:
I'll keep that in mind for future PRs, for now I added a new commit because it makes more sense with the suggestion of @costisin in this PR.